src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -620,6 +620,11 @@ public virtual NamedTypeSymbol TupleUnderlyingType
         public virtual ImmutableArray<string> TupleElementNames => default(ImmutableArray<string>);
```

Comment:
That's a good question. I'm ambivalent, as it is redundant but convenient. 
I'll discuss with Vlad and Aleksey.